src/roles/vengefulghost.py

```diff
@@ -105,28 +105,15 @@ def on_del_player(evt, var, user, nickrole, nicktpls, death_triggers):
         user.send(messages["vengeful_turn"].format(GHOSTS[user]))
         debuglog(user.nick, "(vengeful ghost) TRIGGER", GHOSTS[user])
 
-@event_listener("rename_player")
```

Comment:
Missing swap_player listener (not sure it does anything with VG right now since they're dead and we still have all of those dead player user tracking bugs, but couldn't hurt to add it in for when said bugs get fixed)